crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -971,3 +971,31 @@ pub fn transpose_config(curr: &Node) -> Vec<i64> {
 
     perm
 }
+
+pub fn squeeze_config(curr: &Node) -> Vec<i64> {
+    for (key, value) in curr.attrs.iter() {
+        match key.as_str() {
+            "axes" => return value.clone().into_i64s(),
+            _ => {}
+        }
+    }
+
+    assert!(
+        !curr.inputs.is_empty(),
+        "Squeeze: axes tensor must be present"
+    );
+
+    let input_value = &curr.inputs[1];
+
+    match &curr.inputs[1].ty {
+        ArgType::Tensor(tensor) => {
+            assert_eq!(tensor.dim, 1, "Squeeze: axes tensor must be 1D");
+            if let Some(Data::Int64s(shape)) = input_value.value.as_ref() {
+                shape.clone()
+            } else {
+                panic!("Tensor data type must be int64")
+            }
+        }
+        _ => panic!("Arg for squeeze must be tensor or scalar"),
+    }
```

Comment:
`axes` can contain negative values, which means counting dimensions from the back (see [squeeze spec](https://onnx.ai/onnx/operators/onnx__Squeeze.html#inputs)), and Burn squeeze only supports positive values (see [doc](https://burn.dev/docs/burn/prelude/struct.Tensor.html#method.squeeze)). So we should account for this.

We are already doing this for one dimension (see [code](https://github.com/tracel-ai/burn/blob/main/crates/burn-import/src/onnx/op_configuration.rs#L335-L338)) for `gather_config`, so you can see the logic. We need to do this for all items in `axes`.